manifests/params.pp

```diff
@@ -161,7 +163,7 @@
       $client_nfsv4_services      = undef
       $server_nfsv4_servicehelper = undef
       $server_service_name        = undef
-      notice("\"${module_name}\" provides no service parameters for OS family \"${::osfamily}\"")
+      fail("\"${module_name}\" provides no service parameters for OS family \"${::osfamily}\"")
```

Comment:
Could you please leave the notice, so someone who is using a non supported OS could pass the values as parameters to get this module work.